<launch>
  <!-- launch file arguments -->
  <arg name="ip_address" default="192.168.0.20" />
   
  <!-- netft node -->
  <node name="netft" pkg="netft_rdt_driver" type="netft_node" output="screen">
    <param name="ip_address" type="string" value="$(arg ip_address)"/>
    <param name="sensor_name" type="string" value="netft"/>
    <param name="frame_id" type="string" value="end_effector"/>
    <param name="publish_rate" type="double" value="1000.0"/>
  </node>
  
</launch>
